### DISCUSSION TITLE
Compatible with Elementor

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-checklists#693] [Feature Requests]
Currently the checklist requirement can be bypass if the user using Elementor. It should be good if our checklist also affecting Elementor

Original request: https://secure.helpscout.net/conversation/2391821178/211662?folderId=6932143

**Comment by olatechpro** [comment]
Closing this for https://github.com/publishpress/PublishPress-Checklists/issues/504

**Comment by olatechpro** [comment]
@rizaardiyanto1412 @stevejburge  I re-opened this back as i realised it's not same as the closed issue.

When editing with elementor, here's the UI. So,  how do you think the checklists will work?

![ele](https://github.com/publishpress/PublishPress-Checklists/assets/46832636/0da7134b-8f09-4128-aaaa-7a238c9e0ef2)

**Comment by stevejburge** [comment]
@olatechpro It never will. We can ignore Elementor
